serverless.yml

```diff
@@ -27,12 +27,12 @@ provider:
            - logs:CreateLogStream
            - logs:CreateLogGroup
           Resource:
-           - "arn:aws:logs:${AWS::Region}:${env:ACCOUNT}:log-group:/aws/lambda/cloudflare-lambda-Function-*:*"
+           - Fn::Sub: "arn:aws:logs:${AWS::Region}:${env:ACCOUNT}:log-group:/aws/lambda/cloudflare-lambda-Function-*:*"
```

Comment:
I'm a bit hazy on this syntax.
Is it similar to how we do Subbing via cloudformation?
I see this in their [docs](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference-sub.html)

```yml
Fn::Sub:
  - String
  - Var1Name: Var1Value
    Var2Name: Var2Value
```

Reply:
My understanding is that we're using raw cloudformation in this serverless config, so the yaml cloudformation is exactly the syntax to use. If you look toward the middle of the doc you linked, you can see this section, which is how we're using it here:

Fn::Sub without a mapping
The following example uses Fn::Sub with the AWS::Region and AWS::AccountId pseudo parameters and the vpc resource logical ID to create an Amazon Resource Name (ARN) for a VPC.

Reply:
I think I'm missing something. That syntax example that they have looks different?
```yml
!Sub 'arn:aws:ec2:${AWS::Region}:${AWS::AccountId}:vpc/${vpc}'
```

Unless I'm looking at the wrong thing, which is entirely possible >.>

Reply:
I noticed that too when I looked at your link. I presume both `!Sub` and `Fn::Sub` are accepted syntax because we're already using `Fn::Sub` further down in the Serverless yaml.

Reply:
Cool! Works for me